vhdbuilder/packer/install-dependencies.sh

```diff
@@ -495,6 +495,7 @@ MULTI_ARCH_KUBE_BINARY_VERSIONS="
 1.23.8-hotfix.20220620
 1.24.0-hotfix.20220615
 1.24.3
+1.25.0
 "
 
```

Comment:
`crictl` binary for k8s 1.25.0 needs to be cached in the VHD as well. 
The `1.25.0` version needs to be added in line 150 of `install-dependencies.sh` and it needs to be added to crictl version in components.json as well (line 469).

Reply:
Other parts of the PR look good

Reply:
And we also want to confirm that the VHD builds succeed (might need a retry here).

Reply:
crictl 1.25 added